release: v1.37.0 (#4421)
apollographql/router: .changesets/docs_bnjjj_fix_4342.md

### Document `experimental_when_header` logging configuration option ([Issue #4342](https://github.com/apollographql/router/issues/4342))

> In the router v1.35.0, the experimental logging configuration option `experimental_when_header` was supported but not documented. Its documentation has been added [here](https://apollographql.com/docs/router/configuration/telemetry/exporters/logging/overview#requestresponse-logging).

Author: ```suggestion
In router v1.35.0, we introduced the¬ experimental logging configuration option `experimental_when_header` without documentation. Its documentation has been added [here](https://apollographql.com/docs/router/configuration/telemetry/exporters/logging/overview#requestresponse-logging).
```
